accname/aria-owns.html

```diff
@@ -0,0 +1,236 @@
+<!doctype html>
+<html>
+
+<head>
+  <meta charset="utf-8">
+  <title>Accname tests for assorted combinations of aria-owns, aria-hidden, and host language hiding techniques</title>
+  <script src="/resources/testharness.js"></script>
+  <script src="/resources/testharnessreport.js"></script>
+  <script src="/resources/testdriver.js"></script>
+  <script src="/resources/testdriver-vendor.js"></script>
+  <script src="/resources/testdriver-actions.js"></script>
+  <script src="/wai-aria/scripts/aria-utils.js"></script>
+  <style>
+    [aria-hidden="true"] {
+      outline: 3px dashed darkred;
+    }
+
+    [id^=child] {
+      font-weight: bold;
+      background-color: yellow;
+    }
+  </style>
+</head>
+
+<body>
+
+  <h1>Accname tests for assorted combinations of <code>aria-owns</code>, <code>aria-hidden</code>, and host language hiding techniques</h1>
+  <p><span aria-hidden="true">Text enclosed in a dashed red outline</span> is <code>aria-hidden="true"</code>.</p>
+  <p><span id="child-simulated">Bold text with a yellow background</span> is the target of another element’s <code>aria-owns</code>.</p>
+
+  <h2>Child is moved, nothing is hidden</h2>
+  <p>Expected: <q>I love cats and dogs I like birds</q></p>
```

Comment:
This PR warps my brain. I think we might need a working session to review the intentions here. A few things that might make it easier to review are:

- expected values that are more relevant to the function than random cat/dog/bird strings... Perhaps "[rendered contents owning][rendered owned contents that is child of aria-hidden parent]" or something that indicates the expectation and why you expect it.
- more functional or less contrived examples (to the extent possible), that should obviously work according to an authors intention.